import/import.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I suggest remove `scripts/prepare-data.sh` if this script works better.

Reply:
Done